src/libraries/System.Text.Json/ref/System.Text.Json.cs

```diff
@@ -771,6 +771,7 @@ public abstract partial class JsonNode
         [System.CLSCompliantAttribute(false)]
         public static implicit operator System.Text.Json.Nodes.JsonNode (ulong value) { throw null; }
         public static System.Text.Json.Nodes.JsonNode? Parse(System.IO.Stream utf8Json, System.Text.Json.Nodes.JsonNodeOptions? nodeOptions = default(System.Text.Json.Nodes.JsonNodeOptions?), System.Text.Json.JsonDocumentOptions documentOptions = default(System.Text.Json.JsonDocumentOptions)) { throw null; }
+        public static System.Threading.Tasks.Task<System.Text.Json.Nodes.JsonNode?> ParseAsync(System.IO.Stream utf8Json, System.Text.Json.Nodes.JsonNodeOptions? nodeOptions = default(System.Text.Json.Nodes.JsonNodeOptions?), System.Text.Json.JsonDocumentOptions documentOptions = default(System.Text.Json.JsonDocumentOptions), System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
```

Comment:
I added this line manually, but I have a suspicion that there is a tool for automatically generating such refs. Am I right?

Reply:
https://github.com/dotnet/runtime/blob/main/docs/coding-guidelines/updating-ref-source.md

Reply:
Manual editing is also fine, just make sure alphanumeric ordering of APIs is preserved.